spec/pg/decoders/range_decoder_spec.cr

```diff
@@ -0,0 +1,10 @@
+require "../../spec_helper"
```

Comment:
I created a new spec file to help organize the range specs outside of the regular specs - it looked like that's what happened with `array_decoder_spec.cr`
I guess if this turns into multiple structs for the different ranges, then maybe the decoder ought to be extracted as well?

Reply:
Yeah I don’t have any fully thought out reason, but it kinda makes sense since there will be a lot of related decoders for the ranges.